docs/src/terminology.md

```diff
@@ -67,6 +67,17 @@ A measure of the network confirmation for the [block](#block).
 
 A set of [validators](#validator) maintaining a single [ledger](#ledger).
 
+## cluster restart
+When a [cluster](#cluster) stops finalizing blocks for a prolonged period of
+time, as determined by social consensus amongst the validator operators, a
```

Comment:
`stops finalizing blocks for a prolonged period of time` can be misleading because it' doesn't necessarily have to be a long time. Maybe `due to an unrecoverable set of failures`

Reply:
Done